ci-operator/config/openshift/special-resource-operator/openshift-special-resource-operator-master.yaml

```diff
@@ -20,7 +20,29 @@ images:
       as:
       - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.10
   to: special-resource-rhel8-operator
+- dockerfile_literal: |
+    FROM src
+    RUN ["yum", "update", "-y"]
+    RUN ["yum", "install", "-y", "ShellCheck"]
+    RUN mkdir /tmp/helm && \
+        cd /tmp/helm && \
+        curl https://get.helm.sh/helm-v3.6.0-linux-amd64.tar.gz -o helm.tar.gz && \
+        tar xvfpz helm.tar.gz && \
+        mv linux-amd64/helm /usr/local/bin && \
+        chmod +x /usr/local/bin/helm && \
+        rm -rf /tmp/helm
+    RUN mkdir /tmp/oc && \
+        cd /tmp/oc && \
+        curl -L https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/linux/oc.tar.gz -o oc.tar.gz && \
+        tar xzvf oc.tar.gz && \
+        mv kubectl /usr/local/bin/ && \
+        cd - && \
+        rm -rf /tmp/oc
+  from: src
+  to: test-bin
```

Comment:
Is there a way to re-use that image that was already defined [here](https://github.com/openshift/release/blob/4da87f99dd2cc79613f8b669d86c2ace36539e6b/ci-operator/config/openshift-psap/special-resource-operator/openshift-psap-special-resource-operator-master.yaml#L9-L28)?

It will be better to not need to add any new dependency package at 2 different places.